src/transformers/pipelines.py

```diff
@@ -1206,6 +1285,193 @@ def __call__(
             return results
 
 
+class TranslationPipeline(Pipeline):
+    """
+    Translates from one language to another.
+
+    Usage::
+
+        en_fr_translator = pipeline("translation", from_lng="English", to_lng="French")
```

Comment:
For now, I would probably like to have it like this maybe (user-facing):
```python
nlp = pipeline("translate_en_to_fr")
nlp("How old are you?")
```
What do you think?

(not a big fan of user-facing abbreviations like `from_lng` but it's not the main reason for my proposable just a side note)

Reply:
related opinion: Either way, we will require users to know some magical strings like "en" or "English". Since these strings won't autocomplete in IDEs, we should document and error message well.

[ISO 639-1 codes](https://www.loc.gov/standards/iso639-2/php/code_list.php)